lib/flipper/ui/util.rb

```diff
@@ -11,6 +11,10 @@ def self.blank?(str)
       def self.titleize(str)
         str.to_s.split('_').map(&:capitalize).join(' ')
       end
+
+      def self.descriptions
```

Comment:
Having this here means once the process has booted and set the descriptions there is no changing them without a restart or redeploy, right? The first part of descriptions_source is allowing mysql or whatever to set the descriptions. The second part, that I think is missing from this PR is allowing people to add descriptions anytime and having them show up, without needing to restart processes or redeploy their app. Adding a row to mysql should make the description just show up, but I think this would mean that they would have to restart to force a reload. 

What are your thoughts on that? Also, what are your thoughts on moving to a block for loading and putting that in the actions instead of here?

Reply:
> Having this here means once the process has booted and set the descriptions there is no changing them without a restart or redeploy, right? 

I really hadn't think of that up until this point. I wasn't aware of that to be honest. For sure, let's move the block call to the `actions` module for both `features` and `feature` resources 🚀